app/models/scheme.rb

```diff
@@ -2,7 +2,7 @@ class Scheme < ApplicationRecord
   belongs_to :owning_organisation, class_name: "Organisation"
   belongs_to :managing_organisation, optional: true, class_name: "Organisation"
   has_many :locations, dependent: :delete_all
-  has_many :case_logs, dependent: :delete_all
+  has_many :case_logs, class_name: "LettingsLog", dependent: :delete_all
```

Comment:
`has_many :lettings_logs`?

Reply:
I haven't gotten around to changing this file yet but will be sure I make this change 😄